### ISSUE TITLE
[Request] Migrate to Issue forms

### ISSUE THREAD

**Issue by TheEvilSkeleton** [bottlesdevs/Bottles#1187]
**Is your feature request related to a problem? Please describe.**

At the moment, we are using issue templates, where filing issues requires basic knowledge of Markdown, and can be hard to parse depending on the person.

**Describe the solution you'd like**
GitHub recently introduced [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms). It'd be best to migrate to it. This will facilitate everyone.

**Describe alternatives you've considered**

Using HTML for comments (`<!-- -->`), but this assumes the issuer understands HTML, which is something we should be avoiding.

**Additional context**

This is how it looks like:
![1](https://docs.github.com/assets/cb-61019/images/help/repository/sample-issue-form.png)

If you want to see it in person, you can take a look at [Flatpak's issue tracker](https://github.com/flatpak/flatpak/issues/new/choose).

**Comment by mirkobrombin** [comment]
Woo, nice! I've missed this

**Comment by TheEvilSkeleton** [comment]
I'm going to close this because it has been implemented. Thanks!

**Comment by mirkobrombin** [comment]
Thanks to you!
